docs/src/test/scala/docs/http/scaladsl/server/FullSpecs2TestKitExampleSpec.scala

```diff
@@ -53,8 +53,8 @@ class FullSpecs2TestKitExampleSpec extends Specification with Specs2RouteTest {
     "return a MethodNotAllowed error for PUT requests to the root path" in {
       // tests:
       Put() ~> Route.seal(smallRoute) ~> check {
-        status === StatusCodes.MethodNotAllowed
-        responseAs[String] === "HTTP method not allowed, supported methods: GET"
+        status shouldEqual StatusCodes.MethodNotAllowed
```

Comment:
Yes, that works. I don't have a strong opinion and mainly wanted to question the choice of wording in #1136 and the commit message in that the current assertions are not wrong and actually works as expected.